index.html

```diff
@@ -81,35 +77,37 @@ <h1>I'm Adeyemi Glad to see you!</h1>
             </p>
           </div>
           <div class="Social-media">
-            <div class="socialText"> 
+            <div class="media-Container"> 
               <div class="socioText">
                 <p>LET'S CONNECT</p>
               </div>
             </div>
             <div class="social-media-icon">
-              <li>
-                <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
-                <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin icon.png" alt="Linkedin"></a>
-                <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
-                <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
-                <a class="handwe" href="#github.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
-              </li>
+              <ul>
+                <li>
+                  <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a>
+                  <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" 
+                  alt="Linkedin"></a>
+                  <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
+                  <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
+                  <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
+                </li>  
+              </ul>            
```

Comment:
- [ ] Kindly as requested by the previous reviewer, I think it would be better  to the opening and closing ```li``` tag on every icon. as the [HTML ```<ul>``` tag](https://www.w3docs.com/learn-html/html-ul-tag.html) is used for specifying an unordered list, which groups a collection of items having no numerical order and Each element of an unordered list is declared inside the ```<li>``` and ```</li>``` :+1: 

I hope the following codes given by the previous code reviewer will work :100: 
```
<ul>
     <li>
          <a class="Twitter" href="#Twitter.com/McEmarkees" target="_blank" ><img src="./images/Twitter.png" alt="Twitter"></a >
     </li>
     <li>
          <a class="Linkedin-Icon" href="#Linkined.com/McEmarkees" target="_blank" ><img src="./images/Linkedin_icon.png" 
                alt="Linkedin"></a>
     </li>
     <li>
          <a class="McList" href="#Mclist.com/McEmarkees" target="_blank" ><img src="./images/McList.png" alt="Linkedin"></a>
     </li>
     <li>
          <a class="gitHub" href="#github.com/Emarkees" target="_blank" ><img src="./images/hangama.png" alt="GitHub"></a>
     </li>
     <li>
          <a class="handwe" href="#angelist.com/Emarkees" target="_blank" ><img src="./images/handwe.png" alt="GitHub"></a>
     </li>
</ul>